02_JavaScript_Fundamentals/16_comparisonoperators.js

```diff
@@ -0,0 +1,104 @@
+// if (something) {
+//   do something
+// } else {
+//   do something
+// }
+
+const id = 100;
+
+// //EQUAL TO
+// if (id ==100){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+// // NOT EQUAL TO
+// if (id !=101){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+// // EQUAL TO VALUE & TYPE
+
+// if (id === 100){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+// // NOT EQUAL TO VALUE & TYPE
+
+// if (id !== 100){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+
+//Test is undefined
+// if ( typeof id !== 'undefined'){
+//   console.log (`The ID is ${id}`);
+// } else {
+//   console.log ('NO ID');
+// }
+
+//GREATER THAN OR LESS THAN
+
+// if (id>200){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+// if (id<200){
+//   console.log('CORRECT');
+// } else {
+//   console.log ('INCORRECT');
+// }
+
+// IF ELSE
+
+const color = 'Yellow';
+
+// if (color==='Red'){
+//   console.log ('Color is red');
+// } else if (color === 'Blue') {
+//   console.log('Color is Blue');
+// } else {
+//   console.log('Color is not red or blue')
+// }
+
+// LOGICAL OPERATORS
+
+const name = 'Steave';
+const age =25;
+
+///AND &&
+if ( age > 0 && age<12){
```

Comment:
add parenthesis to the relational expression to make the condition unambiguous. 
like this

if ((age > 0) && (age < 12) {
// do something
}

Reply:
also be consistent on the spaces in the relational expression.